compiler/cli.py

```diff
@@ -156,10 +156,7 @@ def main(argv):
     args = parser.parse_args(argv[1:])
 
     # Parse interpreter from stub file that's not available in Starlark
-    interpreter = parse_stub(args.stub_file)
-
-    if args.interpreter:
-        interpreter = args.interpreter
+    interpreter = args.interpreter or parse_stub(args.stub_file)
```

Comment:
Note `parse_stub` also validates `interpreter`. Really it should be broken into `parse_interpreter` and `validate_interpreter`
then this would be:

```python
interpreter = args.interpreter or parse_interpreter(args.stub_file)
validate_interperter(interpreter)
```

or something